source/releasenotes/2024-05-27-pantheon-advanced-page-cache-2-0.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Pantheon Advanced Page Cache 2.0 update
+published_date: "2024-05-27"
```

Comment:
```suggestion
published_date: "2024-05-28"
```

Reply:
change filename too :)

Reply:
Reviewed and no changes from me. Thanks!